Microservices/event-service/src/main/java/app/sportahub/eventservice/model/event/Event.java

```diff
@@ -0,0 +1,61 @@
+package app.sportahub.eventservice.model.event;
+
+import app.sportahub.eventservice.model.BaseEntity;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper=false)
+@SuperBuilder(toBuilder = true, setterPrefix = "with")
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Data
+public class Event extends BaseEntity {
+
+    @NotBlank(message = "Event name must be provided")
+    private String eventName;
+
+    @NotBlank(message = "Event type must be provided")
+    private String eventType;
+
+    @NotBlank(message = "Sport type must be provided")
+    private String sportType;
+
+    @NotNull
+    private Location location;
+
+
+    @NotEmpty(message = "Date must be provided")
+    private LocalDate date;
+
+    @NotBlank(message = "Duration must be provided")
```

Comment:
Can you make it nullable? We don't know yet if there will be durations in all and each of the events. People can choose not to specify a duration.